README.md

```diff
@@ -28,15 +28,15 @@ Future library releases may include built-in re-usable formatters, for outputtin
 ### Prerequisites
 
 * A Mac
-* Xcode 10.1
+* Xcode 10 or above
```

Comment:
This is prerequisites to build PLCR we should add a section for the prerequisites to consume it mentioning iOS, tvOS and macOS min supported versions.